### DISCUSSION TITLE
Monitoring browsing and email

### DISCUSSION THREAD

**Discussion by elsevillismo** [Security-Onion-Solutions/securityonion#9506] [General]
I have worked with arcsight for 4 years. My question is how can I monitor the internet browsing in real time by the hosts? And also the incoming email in real time?

Thanks.

**Comment by Acewiza** [comment]
SO monitors whatever traffic you tell it to, in real time.  The only thing "un" real about it, is how you approach the output.  Personally, I have better things to do than look over people's shoulder browsing the Internet.  Alternatively, I'd recommend proxy logs if you're contemplating some sort of IT web policy enforcement action.
